.github/workflows/pr-e2e-testing.yaml

```
@@ -65,224 +53,87 @@ jobs:
-      # specific slice of the tests on each parallel job.
-      # =================================================================
-      - name: Run E2E Test Slice
-        if: ${{ steps.changed-files.outputs.any_changed == 'true' }}
-        run: make run-e2e-test VERBOSE=true # use DEBUG_VM_CONSOLE=1 to see the VM console output
-        env:
-          GINKGO_LABEL_FILTER: ${{ inputs.label_filter || 'sanity' }}
-          GINKGO_TOTAL_NODES: ${{ env.GINKGO_TOTAL_NODES }}
-          GINKGO_NODE: ${{ matrix.ginkgo_node }}
+  build-flightctl-cli:
+    needs: build-images-and-charts
+    if: needs.compute-tag.outputs.any_changed == 'true'
```

actionlint (1.7.9)

property "compute-tag" is not defined in object type {build-images-and-charts: {outputs: {cli_git_tag: string; git_tag: string}; result: string}}

(expression)